theseus/geometry/manifold.py

```diff
@@ -45,7 +45,7 @@ def __init__(
                 tensor = self._check_tensor(tensor, strict)
             else:
                 warnings.warn(
-                    f"functorch is enabled and tensor is not checked "
+                    f"Lie group consistency checks are disabled "
```

Comment:
Lie group -> Manifold? Since the class is Manifold.

Reply:
Good catch. Fixed.